pkgs/by-name/la/laravel/package.nix

```
@@ -3,28 +3,37 @@
   fetchFromGitHub,
   makeWrapper,
   php,
+  nodejs ? nodejs,
```

```suggestion
  nodejs,
```
that is not doing what you think it is